sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -45,16 +45,81 @@ function throwMissingEnvironmentVariablesError(envVars: EnvVarKeys[]): void {
   });
 }
 
-function getEnvVarValue(name: string): string | undefined {
+function getEnvVarValue(name: string, forBrowser?: boolean): string | undefined {
```

Comment:
Since we already have access to `isNode`, do we have to have `forBrowser` being passed as an argument here or for `getEnvVars`?

Reply:
yes, it's for marking which environment variables have `_BROWSER` in the key, so this is not true always when in browser. It's only for those specific env vars (such as entities) that need this to be passed in.

Reply:
So here it definitely needs to be passed in. But for in case of `getEnvVars()` - yes, it can be inferred from isNode. 

But based on the issue I'm seeing, I'm concerned about getting the code to work correctly. It's unclear why the entities are getting resolved to undefined. On troubleshooting, found that getEnvVars()  is invoked 3 times for running one test case. First one evaluated to node specific values, and the last one used for actual test run got evaluated to undefined. 

So it's better to keep it as a parameter that we pass in until this issue is clear, resolved.
And possibly, streamline it's usage/design to be better (in before each of tests, for instance)

Reply:
Discussed offline, issue was with entity name not getting computed correctly in later changes. Fixed and addressed the use of `forBrowser` flag as we are clear with what's happening with `getEnvVars()`.
Latest commit also makes the building of browser resources depend on CLEAN_NAMESPACE flag value.